TCMP/BasicNFC-CommandFamily/BasicNFCCommandResolver.swift

```diff
@@ -77,4 +77,25 @@ import Foundation
         try parsedMessage.parsePayload(payload: message.payload)
         return parsedMessage
     }
+    
+    @objc public func getNdefTextPayloadJSON(ndefResponse : NDEFFoundResponse) -> String? {
+        if ndefResponse.ndefMessage.count > 7 {
+            // return the full block
+            let ndefTextBytes = Data(ndefResponse.ndefMessage[1...ndefResponse.ndefMessage.count-1])
```

Comment:
These indices will not get you the text payload you're after. Although I can't incorporate this into the resolver, if you wanted to make this function in your Nativescript app, it should slice from index 7 to the end.  This assumes a lot about the format of the NDEF text record, but as we've discussed this method is simple and naive.  I recommend you pass the entire NDEF message as JSON to your listener and use [NdefJS ](https://github.com/TapTrack/NdefJs) in your Nativescript app to properly parse out the text.